ARK Server Manager/Lib/Model/ServerProfile.cs

```diff
@@ -1889,5 +1896,25 @@ public void ResetOverrideMaxExperiencePointsDino()
         {
             OverrideMaxExperiencePointsDino = GameData.DEFAULT_MAX_EXPERIENCE_POINTS_DINO;
         }
+
+        // section reset methods
+        public void ResetChatAndNotificationSection()
+        {
+            EnableGlobalVoiceChat = GameData.DEFAULT_ENABLE_GLOBAL_VOICE_CHAT;
```

Comment:
If I use the property metadata, I will not have to define constants, unless you want to. The Properties can remain as previously coded. I have no idea which would be the better option to pursue.